sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,393 @@
+- Added OpenCL version check for independent forward progress query. intel/llvm#15872
+- Fixed the bug where queue creation would fail on a system with multiple AMD GPU devices. intel/llvm#15964
+- Aligned checks performed in `is_compatible` implementation with the checks in the standard image selection path. intel/llvm#16060
+- Fixed multi-device support for persistent cache. intel/llvm#16056
+- The barrier dependency for out-of-order profiling tags has been fixed to ensure future work is correctly sequenced relative to the start/end of the profiling tag.  intel/llvm#16112
+- Now `online_compiler::compile` is supported when compiled with pre-C++11 ABI. intel/llvm#16179
+- Exceptions are now correctly thrown for `info::device::preferred_interop_user_sync` and `info::device::profile` when the backend is not OpenCL, in accordance with the SYCL 2020 specification. intel/llvm#16171
+- Now `sycl::opencl::has_extension` compiled with pre-C++11 ABI is supported. intel/llvm#16176
+- Dependencies of empty command groups are now honored. intel/llvm#16203
+
+### Documentation
+
+- Fixed bindless image specification examples. intel/llvm#15726
+- Fixed [`sycl_ext_oneapi_enqueue_functions`](https://github.com/intel/llvm/blob/07942fce8af128b35f5e531d9e4f79ffb4c7febd/sycl/doc/extensions/experimental/sycl_ext_oneapi_enqueue_functions.asciidoc) extension to pass handler by reference to align with the implementation. intel/llvm#15898
+
+### SYCLcompat
+
+- Converted error names to lower case to avoid conflicts with macros. intel/llvm#15373
+- Moved  `memcpy`, `memset`, `free`, `fill` functions into anonymous namespace to prevent clashes. intel/llvm#15446
+
+## Known Issues
+
+- On Windows, the Unified Runtime's Level Zero leak check does not work correctly with
+  the default contexts on Windows. This is because on Windows the release
+  of the plugin DLLs races against the release of static global variables
+  (like the default context).
+- Intel Graphic Compiler's Vector Compute backend does not support O0 code and
+  often gets miscompiled, produces wrong answers and crashes. This issue directly
+  affects ESIMD code at O0. As a temporary workaround, we have optimize ESIMD code
+  even in O0 mode.
+  [00749b1e8](https://github.com/intel/llvm/commit/00749b1e8e3085acfdc63108f073a255842533e2)
+- When using `sycl_ext_oneapi_matrix` extension it is important for some
+  devices to use the sm version (Compute Capability) corresponding to the device
+  that will run the program, i.e. use `-fsycl-targets=nvidia_gpu_sm_xx` during
+  compilation. This particularly affects matrix operations using `half` data
+  type. For more information on this issue consult with
+  https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#wmma-restrictions
+- [new] C/C++ math built-ins (like `exp` or `tanh`) can return incorrect
```

Comment:
@AlexeySachkov Can you please help to understand if these PRs intel/llvm#15489 intel/llvm#15672 intel/llvm#15980 intel/llvm#15808 fixed this issue? If I get it right from one of the PR descriptions, `tanh` problem is caused by problem in msvc headers. And considering that several fixes are landed for `exp`, I am not sure if this issue is still relevant.

Reply:
It's an MSVC issue actually, but we still use the old version, therefore this test case is skipped, see: https://github.com/intel/llvm/issues/16950
So from SYCL POV we addressed all issues with exp.